Cargo.toml

```diff
@@ -101,3 +101,7 @@ harness = false
 [[bench]]
 name = "mantis"
 harness = false
+
+[[bench]]
+name = "nixpkgs"
```

Comment:
This is not a really descriptive name, if not deceptive. I would insist on it benchmarking typechecking, like`typecheck-nixpkgs-lib` or something.